apps/studio/src/routes/editor/WebviewArea/Frame.tsx

```diff
@@ -24,6 +27,7 @@ const Frame = observer(
         const RETRY_TIMEOUT = 3000;
         const DOM_FAILED_DELAY = 3000;
         const editorEngine = useEditorEngine();
+        const projectManager = useProjectsManager().project;
```

Comment:
@drfarrell fyi, this is the easier way to get the current running project. From there you can get the path